CDP4Common/CDP4Common.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;netstandard1.6;netstandard2.0</TargetFrameworks>
     <Company>RHEA System S.A.</Company>
     <Title>CDP4Common Community Edition</Title>
-    <Version>4.0.0</Version>
+    <VersionPrefix>4.0.1</VersionPrefix>

```

Comment:
Because we create pre-release on DevOps that need a "higher" version than the last one.
That is done by using a prefix and a suffix part to construct the version number.
In the "nuget package world" 4.0.0-prefix is a lower version than 4.0.0.
4.0.0 is a release, so for the next pre-release the version has to be higher.
I choose to add a +1 to the patch number. 
Indeed we can update the minor, or even the major version number when we create a release.